src/Former/Traits/Field.php

```diff
@@ -132,7 +132,7 @@ public function __call($method, $parameters)
 		}
 
 		// Redirect calls to the Control Group
-		if (method_exists($this->group, $method) or Str::startsWith($method, 'onGroup')) {
+		if (method_exists($this->group ?: '', $method) or Str::startsWith($method, 'onGroup')) {
```

Comment:
Wouldn't it make more sense to do it like this? No reason to check for an empty method name to exists because we know it will not exist right?

```suggestion
		if ((!empty($this->group) && method_exists($this->group, $method)) or Str::startsWith($method, 'onGroup')) {
```

Reply:
Either works. Your version does the same, just a bit more expressive which might be a better. Possibly has better bc support with older PHP versions.